app/viewmodels/cya/CYAUpdateCaseSummaryListHelper.scala

```diff
@@ -71,8 +71,8 @@ trait CYAUpdateCaseSummaryListHelper {
         keyText = Text(messages("updateCase.cya.moreDocumentation")),
         valueContent = Text(anyMoreDocumentation),
         action = Some(ActionItemHelper.createChangeActionItem(
-          "url",
-          ""
+          controllers.routes.MoreDocumentationController.onLoad().url,
+          messages("updateCase.cya.moreDocumentation")
```

Comment:
I think this needs to be a new message "updateCase.cya.moreDocumentation.change" with the text "Change add more documentation?"